internal/kv/example_test.go

```diff
@@ -0,0 +1,99 @@
+package kv_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/brexhq/substation/config"
+	"github.com/brexhq/substation/internal/kv"
+)
+
+func Example_memory() {
```

Comment:
Are there instructions for how someone should use this example test / what should be filled in (TODO)?

Reply:
Not really, this is meant as an example for developers and not users who interact through configurations. Once we've merged this the user-facing documentation would be added to substation.readme.io, both as a [processor](https://substation.readme.io/docs/processors) and likely in a new section that describes the different types of KV stores.